h2o-core/src/main/java/water/H2ONode.java

```diff
@@ -41,7 +39,8 @@
     
   transient private short _timestamp;
   transient private boolean _removed_from_cloud;
-  transient private volatile boolean _accessed_local_dkv; // Did this remote node ever accessed the local portion of DKV?   
+  transient private boolean _accessed_local_dkv; // Did this remote node ever accessed the local portion of DKV?
+  transient private IcedHashMap<String, Object> _configuration;
```

Comment:
this is transient field - not serialized, what is the purpose of the _configuration field? it is not sent over the wire

Reply:
It keeps specific node configuration information in a hash map before they are written (along with some other information) to overall table Model#_reproducibility_information_table in createNodeInformationTable()